documentation/source/user_section/tutorials/spinalcord-segmentation/sct_propseg-example-t2.rst

```diff
@@ -0,0 +1,58 @@
+Hands-on Example: T2
+####################
+
+Run the following command to process the image:
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t2_qc.png
+  :align: right
+  :figwidth: 40%
+
+  The QC report for the segmented image
+
+.. code:: sh
+
+   sct_propseg -i t2.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Input image
+   - ``-c`` : Contrast of the input image
+   - ``-qc`` : Directory for Quality Control reporting. QC reports allow us to evaluate the segmentation slice-by-slice
+
+:Output files/folders:
+   - ``t2_seg.nii.gz`` : 3D binary mask of the segmented spinal cord
+
+During execution, the script will provide status updates as it progress through its various stages.
+
+
+Inspecting the results using QC Reports
+***************************************
+
+When complete, the script will output a command to inspect the results. (**Note:** The exact filepath will vary depending on your filesystem.)
+
+.. code:: sh
+
+   Use the following command to see the results in a browser:
+   xdg-open "sct_course_london20/single_subject/data/t2/qc_singleSubj/index.html"
```

Comment:
will not work on macOs, which uses `open` instead of `xdg-open`. Do we want to give the user both options?

Also, the path should be relative to where the user is now (ie: `open/xdg-open qc_singleSubj/index.html`)